### PR TITLE
Restructure account commands to account subcommand

### PR DESCRIPTION
**Pull request by ellie**
Also edit the history count trigger so it only runs on insert, and not on delete. This will make sure that deletes run super fast 🚀

### CODE REVIEW

File: atuin-server/migrations/20230515221038_trigger-delete-only.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+				new.user_id, 
+				(select count(1) from history where user_id = new.user_id)
+			);
+		end if;
+	end if;
+
+	return NEW; -- this is actually ignored for an after trigger, but oh well
+end;
+$func$
+language plpgsql volatile -- pldfplplpflh
```
Comment (conradludgate) [inline atuin-server/migrations/20230515221038_trigger-delete-only.sql:24]: Did you sneeze?

**Review by conradludgate — COMMENTED** [review]
While this works as it did before, I realise that we don't count actual history delete requests here. Should we add a trigger on update? I think users might be confused if the remote count doesn't decrease when they delete

**Comment by ellie** [comment]
> While this works as it did before, I realise that we don't count actual history delete requests here. Should we add a trigger on update? I think users might be confused if the remote count doesn't decrease when they delete

I wish :( If we actually update the remote count when rows are soft deleted, sync won't work. It's append-only, afterall, and expects counts to always increase

File: atuin-server/migrations/20230515221038_trigger-delete-only.sql (hunk 1 above)
Comment (ellie) [reply atuin-server/migrations/20230515221038_trigger-delete-only.sql:24]: 🤧

Comment (ellie) [reply atuin-server/migrations/20230515221038_trigger-delete-only.sql:24]: No I just think the language statement is silly

**Comment by ellie** [comment]
We should perhaps offset the deleted items from the remote count before displaying it though

**Review by conradludgate — APPROVED** [review]

**Pull request merged**